src/_SignAssembly.targets

```diff
@@ -0,0 +1,40 @@
+<Project>
+  <!--
+      Conditional strong name signing of netstandard assemblies for better cross-runtime compatibility.
+      
+        Ref: https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/strong-name-signing-exceptions#recommended-action
+     -->
+
+  <!--
+    NOTE: GraphQLParser.Tests depends on netstandard 2.1 for netcore3.1 and net5.0 tests, requiring a signed friend assembly for internals visible to.
+  -->
+  <PropertyGroup Condition="'$(TargetFramework)'=='net6'">
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
```

Comment:
MS assemblies compiled for .NET 6 are still signed.  It's just that .NET Core ignores them.  I would do the same here.

![image](https://user-images.githubusercontent.com/6377684/183159009-f8186af4-ccf7-4398-8486-2844eef806d8.png)

![image](https://user-images.githubusercontent.com/6377684/183159043-fe15901e-1794-47a8-9baf-91b512e3f006.png)

Reply:
I know and I considered always signing too. But then I decided to skip the net6.0 specific build since it was so easy to do.

According to MS then net6.0 will completely ignore if the assembly is signed, making it a waste of space, literally.

I can remove the conditional signing if you prefer?

Reply:
Let’s remove the conditions. It will be a simpler props file and matches what Microsoft does for their packages

Reply:
@anderslaub lmk when done and I can merge

Reply:
@Shane32 Done. I've also moved the SignAssembly prop to directory.build.props so it gets set before loading the project props during build - just so it can be overridden in a project file if this will be needed at some later point for something (not sure why it would be).